syncer/sharding_group.go

```diff
@@ -119,40 +127,28 @@ func NewShardingGroup(sources []string, isSchemaOnly bool) *ShardingGroup {
 
 // Merge merges new sources to exists
 // used cases
-//   * add a new database / table to exists sharding group
+//   * add a new table to exists sharding group
 //   * add new table(s) to parent database's sharding group
-//  if group is un-resolved, we add it in sources and set it true
+//  if group is in sequence sharding, return error directly
 //  othereise add it in source, set it false and increment remain
 func (sg *ShardingGroup) Merge(sources []string) (bool, bool, int, error) {
 	sg.Lock()
 	defer sg.Unlock()
 
-	// need to check whether source is exist? but we maybe re-sync more times
-	isResolving := sg.remain != len(sg.sources)
-	ddls := []string{"create table"}
+	// NOTE: we don't support add shard table when in sequence sharding
+	if sg.meta.InSequenceSharding() {
+		return true, sg.remain <= 0, sg.remain, errors.NotSupportedf("in sequence sharding, can't add table")
```

Comment:
can we including the current DDL and the sequence sharding in the error message? if so, do that in L165 too.